## <span style="color:darkblue"> Problem Set 4: Cconfidence intervals


**Problem set policies:** Please provide concise, clear answers for each question. Note that only writing the result of a calculation (e.g., "SD = 3.3") without explanation is not sufficient. For problems involving `R`, include the code in your solution, along with any plots.
		
Each problem set is due by 11:59pm on the due date; please submit your problem set via gyrd
		
*You are encouraged to discuss problems with other students (and, of course, with the course head and the TAs), but you must write your final answer in your own words. Solutions prepared "in committee" are not acceptable. If you do collaborate with classmates on a problem, please list your collaborators on your solution.*

## Problem 1
### oi_biostat 4.14

Write the null and alternative hypotheses in words and using symbols for each of the following situations.

(a) Since 2008, chain restaurants in California have been required to display calorie counts of each menu item. Prior to menus displaying calorie counts, the average calorie intake of diners at a restaurant was 1100 calories. After calorie counts started to be displayed on menus, a nutritionist collected data on the number of calories consumed at this restaurant from a random sample of diners. Do these data provide convincing evidence of a difference in the average calorie intake of a diners at this restaurant?


Answer:

$H_0:$ $\mu=1100$

$H_A:$ $\mu\neq 1000$


(b) Based on the performance of those who took the GRE exam between July 1, 2004 and June 30, 2007, the average Verbal Reasoning score was calculated to be 462. In 2011 the average verbal score was slightly higher. Do these data provide convincing evidence that the average GRE Verbal Reasoning score has changed since 2004?

Answer: 

$H_0:\mu=462$

$H_A:\mu<462$

## Problem 2 
#### (oi_biostat 4.4) 

In  2013,  the  Pew  Research  Foundation  reported  that  “45%  of  U.S.
adults report that they live with one or more chronic conditions”.
However, this value was based
on a sample, so it may not be a perfect estimate for the population parameter of interest on its own.
The study reported a standard error of about 1.2%, and a normal model may reasonably be used in
this setting. Create a 95% confidence interval for the proportion of U.S. adults who live with one or
more chronic conditions. Also, interpret the confidence interval in the context of the study.

In [2]:
#Computation here
45+1.96*1.2


[1] 47.352

Answer:

Confidence interval is $(42.6\%,47.4\%)$, i.e. we are 95% confident that the percent of the population who live with one or more chronic conditions is between 42.6\% and 47.4\%.

## Problem 3
#### (oi_biostat 4.6)

In 2013, the Pew Research Foundation reported that “45% of U.S. adults report that they live with one or more chronic conditions”, and the standard error for this estimate is 1.2%. Identify each of the following statements as true or false. Provide an explanation to justify each of your answers.

(a) TRUE or FALSE: We can say with certainty that the confidence interval from Exercise 4.4 contains the true percentage of U.S. adults who suffer from a chronic illness.

Answer: False, assuming that complete randomness, there is a 5\% chance that it is not.

b) TRUE or FALSE: If we repeated this study 1,000 times and constructed a 95% confidence interval for each study, then approximately 950 of those confidence intervals would contain the true fraction of U.S. adults who suffer from chronic illnesses.

Answer: True, this is why we constructed the confidence intervals the way we did.

c) TRUE or FALSE:  The poll provides statistically significant evidence (at the α = 0.05 level) that the percentage of U.S. adults who suffer from chronic illnesses is below 50%.

Answer: True, 50>47.4.

d) TRUE or FALSE: Since the standard error is 1.2%, only 1.2% of people in the study communicated uncertainty about their answer.

Answer: False, this is nonsense.

#### Problem 3 (oi_biostat 4.8)

The 2010 General Social Survey asked the question:  “After an average
work day,  about how many hours do you have to relax or pursue activities that you enjoy?" to a
random sample of 1,155 Americans.
A 95% confidence interval for the mean number of hours
spent relaxing or pursuing activities they enjoy is (1.38, 1.92).

a)  Interpret this interval in context of the data.

Answer: We are 95% confindent that the number of hours Joe America has to pursue leasure activities  is between 1.38 and 1.92.

Alternatively, we are 95% confident that we can get at least 1.38 more hours per  out of our average worker. (:

b) Suppose another set of researchers reported a confidence interval with a larger margin of error
based on the same sample of 1,155 Americans. How does their confidence level compare to the
confidence level of the interval stated above?

Answer: They would have a higher confidence level. Wider interval = more confidence.

c) Suppose next year a new survey asking the same question is conducted, and this time the sample
size is 2,500.   Assuming that the population characteristics,  with respect to  how much time
people spend relaxing after work, have not changed much within a year.  How will the margin
of error of the new 95% confidence interval compare to the margin of error of the interval stated
above?

Answer: The margerin of error will be smaller.